style_unify

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+files=$(find src -name "*.[ch]")
+for i in $files
+do
+    echo $i
+    astyle --style=linux -cnpUH -s4 -M120 $i
```

Comment:
I'm very excited about this patches! :+1:

Just a comment about this line, I would recommend to use the long names of the flag, for sake of autodocumentation:

astyle --style=linux --convert-tabs --suffix=none --pad-oper --unpad-paren --pad-header --indent-spaces=4 --max-instatement-indent=120